pkg/printers/internalversion/printers_test.go

```diff
@@ -68,6 +68,73 @@ func TestAllHandlers(t *testing.T) {
 	}
 }
 
+func TestPrintEvent(t *testing.T) {
+	tests := []struct {
+		event    api.Event
+		options  printers.GenerateOptions
+		expected []metav1beta1.TableRow
+	}{
+		// Basic event; no generate options
+		{
+			event: api.Event{
+				Source: api.EventSource{Component: "kubelet"},
+				InvolvedObject: api.ObjectReference{
+					Kind:      "Pod",
+					Name:      "Pod Name",
+					FieldPath: "spec.containers{foo}",
+				},
+				Reason:         "Event Reason",
+				Message:        "Message Data",
+				FirstTimestamp: metav1.Time{Time: time.Now().UTC().AddDate(0, 0, -3)},
+				LastTimestamp:  metav1.Time{Time: time.Now().UTC().AddDate(0, 0, -2)},
+				Count:          6,
+				Type:           api.EventTypeNormal,
+				ObjectMeta:     metav1.ObjectMeta{Name: "event1"},
+			},
+			options: printers.GenerateOptions{},
+			// Columns: Last Seen, Type, Reason, Object, Message
```

Comment:
The headers are created and tested in `tableprinter.go` and `tableprinter_test.go`. Since none the `print` tests create headers, I don't believe we should test them here.

Reply:
Ok